.gitignore

```diff
@@ -0,0 +1,11 @@
+/node_modules
+/public/hot
+/public/storage
```

Comment:
please add public css and js since I will be running npm compiler most of the time